aspnet/migration/rc1-to-rc2.rst

```diff
@@ -17,7 +17,7 @@ For information regarding a complete list of breaking changes in RC2 for ASP.NET
 Creating your web application host
 ----------------------------------
 
-Since ASP.NET Core apps are just console apps, you must define an entry point for your application in ``Program.Main`` that sets up a web host, then tells it to start listening. Below is an example of the startup code for the built-in `Web Application` template in Visual Studio.
+Since ASP.NET Core apps are just console apps, you must define an entry point for your application in ``Program.Main()`` that sets up a web host, then tells it to start listening. Below is an example of the startup code for the built-in `Web Application` template in Visual Studio.
```

Comment:
Since "the entry point" is the subject... "tells" is the correct grammar. To verify, use the standard grammar technique of removing the superfluous verbiage, "for your application in ``Program.Main()`` that sets up a web host" and read the sentence, which is then "the entry point tells it to start listening". 
See the Strunk & White manual for clarification). I can reword for two sentences though.